README.md

```diff
@@ -162,12 +162,29 @@ verified in some [tests](./.ci/clusters).
 
 ## Grafana Dashboards
 
-The Apache Pulsar Helm Chart uses the `kube-prometheus-stack` Helm Chart to deploy Grafana. Dashboards are loaded via a Kubernetes `ConfigMap`. Please see their documentation for loading those dashboards.
+The Apache Pulsar Helm Chart uses the `kube-prometheus-stack` Helm Chart to deploy Grafana.
 
-The `apache/pulsar` GitHub repo contains some dashboards [here](https://github.com/apache/pulsar/tree/master/grafana).
+There are several ways to configure Grafana dashboards. The default `values.yaml` comes examples of Pulsar dashboards which 
+get downloaded from https://github.com/streamnative/apache-pulsar-grafana-dashboard by URL.
```

Comment:
Should this be a hyperlink e.g. `[label][https://xxxx]` for consistency with the below?